configs/ci/roles/ros.json

```
@@ -0,0 +1,35 @@
+{
+    "roles": [
+      {
+        "name": "Resource Optimization Administrator",
```

For consistency across other admin roles, this should be lowercase.
```suggestion
        "name": "Resource Optimization administrator",
```